en/mkdocs.yml

```diff
@@ -62,6 +62,7 @@ nav:
   - API Proxies: api-proxies.md
   - Triggers: triggers.md
   - Scheduled Tasks: scheduled-tasks.md
+  - GraphQL APIs: graphql-api.md
```

Comment:
We also need to add `preview-features.md` under `references` I suppose.

Reply:
It was initially added and then removed because this feature no longer requires to be enabled that way. However, the guide was left without being linked to the navigator because it will be needed for one or more other features as discussed with Miraj.